libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -301,7 +305,17 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @User: Advanced
     // @RebootRequired: True
     AP_GROUPINFO("ALT_CONFIG", 22, AP_BoardConfig, _alt_config, 0),
-    
+
+#if HAL_HAVE_IMU_HEATER
+    // @Param: TEMPMGN_LOW
+    // @DisplayName: hearter temp lower margin
+    // @Description: Arming check will fail if IMU temp is more than this value lower than BRD_IMU_TARGTEMP, 0 disables
+    // @Range: 0 20
```

Comment:
I wonder if maybe the acceptable range should be larger?  Surely 30deg range is also OK in some cases?